src/libs/ascent/ascent.hpp

```diff
@@ -66,13 +66,34 @@ class ASCENT_API Ascent
     conduit::Node  m_info;
 };
 
+// Callback maps
+static std::map<std::string, void (*)(conduit::Node &, conduit::Node &)> m_void_callback_map;
+static std::map<std::string, bool (*)(void)> m_bool_callback_map;
```

Comment:
We should move this to the ascent.cpp file to avoid possible issues with multiple definitions when ascent.hpp is included by users.